tools/cldr-apps/js/src/views/SignCla.vue

```diff
@@ -154,28 +235,73 @@ if (!user) window.location.replace("#///"); // need a user
 
 let readonlyCla = ref(false);
 let needCla = ref(!user?.claSigned);
-let loading = ref(false);
+let loading = ref(true);
 const userName = ref(user?.name);
 const userEmail = ref(user?.email);
 let userEmployer = ref(user?.org);
 let userSign = ref(0);
+/** if set, the Github ID of a successful CLA sign */
+let userGithubSign = ref(null);
+/** if set, the URL to login with Github. */
+let githubLoginUrl = ref(null);
+/** if set, URL to re-login with Github */
+let githubReloginUrl = ref(null);
+/** if set, the Github ID of the current session ID (may not be on CLA list) */
+let githubSessionId = ref(null);
+/** if true, user has clicked "manual sign" */
+let skipGithub = ref(false);
+
 const radioStyle = {
   display: "flex",
   "padding-bottom": "1em",
 };
 
 async function loadData() {
-  if (!user) return;
-  loading.value = true;
-  const { corporate, email, employer, name, readonly, unauthorized, signed } =
-    await cldrCla.getCla();
+  if (!user) {
+    loading.value = false;
+    return;
+  }
+  const githubLoginLoad = cldrAuth.getLoginUrl({
+    intent: "cla",
+    service: "github",
+  });
+  const githubReloginLoad = cldrAuth.getLoginUrl({
+    intent: "cla",
+    service: "github",
+    relogin: true,
+  });
+  const githubSessionLoad = cldrAuth.getGithubIdFromSession();
+  const {
+    corporate,
+    email,
+    employer,
+    name,
+    readonly,
+    unauthorized,
+    signed,
+    github,
+  } = await cldrCla.getCla();
+
+  try {
+    githubLoginUrl.value = await githubLoginLoad;
+    githubReloginUrl.value = await githubReloginLoad;
+  } catch (e) {
+    console.error("could not getLoginUrl", e);
+  }
+  try {
+    githubSessionId.value = await githubSessionLoad;
+  } catch (e) {
+    console.error("could not getGithubIdFromSession", e);
+  }
+
   loading.value = false;
   if (unauthorized || !signed) return;
   readonlyCla.value = readonly;
   userName.value = name;
   userEmail.value = email;
   userEmployer.value = employer;
   userSign.value = corporate ? 1 : 2;
```

Comment:
Are these numbers for userSign explained somewhere? Seems like named constants might help

Reply:
They are only used in the radio buttons, I'm not sure if it takes a constant